### PR TITLE
Document "Print Debug GC" command and OCAMLRUNPARAM environment variable

### PR DESCRIPTION
**Pull request by jfehrle**

### CODE REVIEW

**Comment by jfehrle** [comment]
@Zimmi48 Who should review this?

**Review by Zimmi48 — APPROVED** [review]
I don't know anything about this topic but this looks good to me. I call @ppedrot to have a look if he wishes. I will merge in a few days if no comments.

**Comment by Zimmi48** [comment]
Were any of these features added in 8.13? Otherwise, why not put this in the 8.12.1 milestone?

**Comment by jfehrle** [comment]
> Were any of these features added in 8.13? Otherwise, why not put this in the 8.12.1 milestone?

No.  I thought it was up to the release manager to choose what goes into the dot release.

**Comment by Zimmi48** [comment]
> I thought it was up to the release manager to choose what goes into the dot release.

It is, but your choice of a milestone as a PR author or assignee should only be seen as a proposal. Ultimately, the RM can decide to change it.

File: doc/sphinx/practical-tools/coq-commands.rst
Comment (jfehrle) [reply doc/sphinx/practical-tools/coq-commands.rst:125]: Maybe.  But not today.  After a month, the details fade and my mind is filled with details for other work.

I removed the other "todo" but kept this one.

**Comment by Zimmi48** [comment]
@coqbot: merge now

**Pull request merged**
